client/styles/spice.scss

```diff
@@ -0,0 +1,7 @@
+.mockup {
+  color:gray;
+  padding-top:1em;
+  width:100%;
+  text-align: center;
+  font-size: larger;
+}
```

Comment:
Doesn't look like it? It's fine either way though as long as we stick with some convention.